README.md

```diff
@@ -1,10 +1,155 @@
 # Seestar_Alp
 
+There are many things in this repository to allow you to interact with your Seestar in interesting ways.
+
+./device directory has the code for the actual seestar control program. It accepts and returns JSON strings
```

Comment:
./device should be backticked - `./device` - the formatting distinguishes and separates it from the text
Same with all directories below

Reply:
Done